@Types/modules.d.ts

```diff
@@ -16,14 +16,14 @@ declare module 'fast-isnumeric' {
 }
 
 declare class Remarkable {
-    constructor();
+    constructor(options?: any);
```

Comment:
Just realized this `@types` exists for the library, maybe we can do away with our custom wrapper.
https://www.npmjs.com/package/@types/remarkable